fspiop-api/documents/API Definition v2.0-alpha.md

```diff
@@ -2997,8 +2995,8 @@ The callback **PUT /fxQuotes/**_{ID}_ is used to inform the client of a requeste
 | **Name** | **Cardinality** | **Type** | **Description** |
 | --- | --- | --- | --- |
 | **condition** | 0..1 | ilpCondition | The ILP condition for the conversion. |
-| **ilpPacket** | 0..1 | ilpPacket | The ILP Packet containing the amount delivered to the Payee, the ILP Address of the Payee and any other end-to-end data. |
-| **conversionTerms** | 1 | Conversion | The terms under which the FXP will undertake the currency conversion proposed by the requester.  |
+| **ilpPacket** | 0..1 | ilpPacket | The ILP Packet containing the amount delivered to the requestingFSP, the ILP Address of the requesting FSP and any other end-to-end data. |
```

Comment:
I think you used "client" instead of "requesting FSP" earlier.. Below "the FSP that requested the conversion" is used.

Reply:
I think it is the Payee FSP, on reflection...

Reply:
Text has been updated so resolving this